crates/universal-sierra-compiler-api/src/utils.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::Result;
+use camino::Utf8PathBuf;
+use indicatif::{ProgressBar, ProgressStyle};
+use std::env;
+use std::time::Duration;
+
+pub fn spawn_spinner_message(sierra_file_path: &Utf8PathBuf) -> Result<ProgressBar> {
+    let spinner = ProgressBar::new_spinner();
+    spinner.set_style(ProgressStyle::with_template("\n{spinner} {msg}\n")?);
+    spinner.enable_steady_tick(Duration::from_millis(100));
+
+    // Skip printing path when compiling unsaved sierra
+    // which occurs during test execution for some cheatcodes e.g. `replace_bytecode`
+    let message = if is_temp_file(sierra_file_path)? {
+        "Compiling SIERRA to CASM".to_string()
+    } else {
+        format!(
+            "Compiling SIERRA to CASM ({})",
```

Comment:
nit: I think `Sierra` should stay as it was.
```suggestion
        "Compiling Sierra to CASM".to_string()
    } else {
        format!(
            "Compiling Sierra to CASM ({})",
```

Reply:
On second thoughts, these are actually no acronyms, hence we should stick to the forms `Sierra` and `Casm`. This is also how they are mostly referred to in the starknet documentation.